### ISSUE TITLE
Aws node8 depreciation 

### ISSUE THREAD

**Issue by Voulnet** [OWASP/Serverless-Goat#16]
Hi. Will serverless goat be affected by AWS deprecating node8.10 in lambda?
I think deprecation will start in jan 2020

**Comment by tommywo** [comment]
Yes, it is no longer possible to deploy node8.10 lambdas. 
node10.x and node12.x images no longer contain `curl` so i've added it, see #17 
This is significantly more work than downloading the document in nodejs, so it makes it a less plausible solution than using nodejs dependency to do the download.

**Comment by halfluke** [comment]
Yes but can't the AWS image be fixed for nodejs 10/12 as they did for DVSA? Or add instructions on how to deploy it from the fixed pull request?

Thanks

**Comment by franruiz** [comment]
Stuck at the same problem. Do we have any chances this is getting fixed ?

**Comment by Voulnet** [comment]
> 
> 
> Stuck at the same problem. Do we have any chances this is getting fixed ?

Yes @franruiz  there is a version on the serverless store in AWS, look for appsec-serverless-goat, works on my account correctly.
